Xamarin.Essentials/Platform/Platform.uwp.cs

```diff
@@ -7,13 +7,13 @@ namespace Xamarin.Essentials
     public static partial class Platform
     {
         static bool PlatformIsMainThread =>
-            CoreApplication.MainView.CoreWindow.Dispatcher == null;
+            CoreApplication.MainView.CoreWindow.Dispatcher.HasThreadAccess;
```

Comment:
should it be.... CoreApplication.MainView.CoreWindow.Dispatcher?.HasThreadAccess ?? true;

So if it is null then on main thread or has thread access.

Reply:
I don't think it should ever be null... The SO appears to confirm that: https://stackoverflow.com/a/18485317/1117914

Reply:
I guess the question is... is there a corewindow? perhaps this code is in somewhere that it isn't...?

Reply:
good point.